edenai_apis/apis/corticalio/helpers.py

```diff
@@ -0,0 +1,14 @@
+from typing import Dict
+
+from edenai_apis.features.text import KeywordExtractionDataClass, InfosKeywordExtractionDataClass
+
+
+def normalize_keywords(response: Dict):
+    items = [
+        InfosKeywordExtractionDataClass(
+            keyword=keyword["word"],
+            importance="{:.2f}".format(keyword["score"])
+        ) for keyword in response["keywords"]
+    ]
+
+    return KeywordExtractionDataClass(items=items)
```

Comment:
I would actually move this into the corticalio_api.py but if it is the general coding schema it's also fine

Reply:
`corticalio_api.py` is an single point of implementation of all supported features (via abstract base classes) and will potentially get quite huge